FoundationDB.md

```diff
@@ -0,0 +1,63 @@
+## WAL-G for FoundationDB
+
+**Work in progress**
+
+You can use wal-g as a tool for encrypting, compressing FoundationDB backups and push/fetch them to/from storage.
+
+Development
+-----------
+### Installing
+To compile and build the binary for FoundationDB:
+
+Optional:
+
+- To build with libsodium, just set `USE_LIBSODIUM` environment variable.
+- To build with lzo decompressor, just set `USE_LZO` environment variable.
+```
+go get github.com/wal-g/wal-g
+cd $GOPATH/src/github.com/wal-g/wal-g
+make install
+make deps
+make fdb_build
+```
+Users can also install WAL-G by using `make install`. Specifying the GOBIN environment variable before installing allows the user to specify the installation location. On default, `make install` puts the compiled binary in `go/bin`.
+```
+export GOBIN=/usr/local/bin
+cd $GOPATH/src/github.com/wal-g/wal-g
+make install
+make deps
+make fdb_install
+```
+
+Usage
+-----
+
+* ``backup-fetch``
+
+Command for sending backup from storage to stream in order to restore it in the database.
+
+```
+wal-g backup-fetch example_backup
+```
+
+Variable _WALG_STREAM_RESTORE_COMMAND_ is required for use backup-fetch
+(eg. ```TMP_DIR=$(mktemp -d) && chmod 777 $TMP_DIR && tar -xf - -C $TMP_DIR && BACKUP_DIR=$(find $TMP_DIR -mindepth 1 -print -quit) && fdbrestore start -r file://$BACKUP_DIR -w --dest_cluster_file "/etc/foundationdb/fdb.cluster"  1>&2```)
```

Comment:
It would be cool to simplify it.

Reply:
I agree but let's keep it like this for the initial version. I can revisit this later